tests/README.md

```diff
@@ -63,3 +63,8 @@ Cached transactions are stored in `tests/__txcache__` directory in the same stru
 Cached transactions are provided to test fixtures via [TX_CACHE](./__txcache__/index.js) utility.
 
 Missing tx json? use [this tool](./__txcache__/gen-reftx.js) to generate it.
+
+## Websocket cache
```

Comment:
If I understood this right the Transaction and Websocket caches are not caches in the usual sense of eg. networking caches. I.e. they do not act as transparent layer that can forward requests to the 'origin' if the response isn't available locally. Instead they're more like mocking tools that let you supply your own responses. Right?

If this is the case can we maybe not call these "Caches"? When I read "cache" I imagine something that merely *caches* responses to reduce bandwith but definitely not something that mocks the responses.

Reply:
not quite, Transaction cache is called "cache" because i want to keep naming convention with [trezor-firmware](https://github.com/trezor/trezor-firmware/tree/master/tests/txcache). One day those files will be shared via [trezor-common](https://github.com/trezor/connect/issues/781)

Websocket cache are mocked (cached 1:1) responses of `blockchain-link` to reduce unnecessary websocket connection.

In both cases tests with disabled cache (on gitlab CI) will work with exact same values, but dowloaded from the internet

Reply:
I agree _cache_ might not be the greatest naming here (but mock isn't either) but as @szymonlesisz puts it I would keep it to have it the same with FW for now.

Reply:
> Transaction cache is called "cache" because i want to keep naming convention with trezor-firmware. One day those files will be shared via trezor-common

Allright didn't know of this parity.